ci-operator/step-registry/openstack/conf/creatednsrecords/openstack-conf-creatednsrecords-commands.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+export AWS_DEFAULT_REGION=us-east-1
+export AWS_DEFAULT_OUTPUT=json
+export AWS_PROFILE=openshift-ci-infra
+
+EXPIRATION_DATE=$(date -d '4 hours' --iso=minutes --utc)
```

Comment:
what is this for.

Reply:
This should be used to label created resources (FIPs) so that external pruner could remove those. Lets remove that if no external pruner for Openstack resources available

Reply:
removing